go/vt/vtctl/workflow/utils.go

```diff
@@ -120,12 +124,6 @@ func validateNewWorkflow(ctx context.Context, ts *topo.Server, tmc tmclient.Tabl
 					allErrors.RecordError(fmt.Errorf("workflow %s already exists in keyspace %s on tablet %v", workflow, keyspace, primary.Alias))
 					return
 				}
-				for _, stream := range wf.Streams {
```

Comment:
Why was this check deleted?

Reply:
The problem with this check is that we can't start any other workflow if there is one that has been switched but not cancelled/completed. You can start multiple workflows if they are all `running` but the moment you switch traffic of one of them you can't create any more!

Ideally we should be doing these validations in the client based on the current status/attributes of existing workflows. I added one at https://github.com/vitessio/vitess/pull/15546/files#diff-327b08a43697d540e06dfd4e4c4d0cb617dc66f9ad3d74c451b926405c9faae5R1712.